types/atvlegacycontentkit/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
can't we update the header's first line to something like `// Type definitions for non-npm package atvlegacycontentkit 7.7` ? it would be better than disabling this rule here

Reply:
Didn't realise it was so straightforward! Have now pushed a change that will do this.